Description: Update libhdatedocdir to the one used by the debian
Author: Debian Hebrew Packaging Team <debian-hebrew-package@lists.alioth.debian.org>

--- a/Makefile.am
+++ b/Makefile.am
@@ -5,7 +5,7 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libhdate.pc
 
-libhdatedocdir = ${prefix}/share/doc/libhdate
+libhdatedocdir = ${prefix}/share/doc/libhdate1
 libhdatedoc_DATA = \
 	README\
 	COPYING\
